README.md

```diff
@@ -38,13 +38,12 @@ Prerequisites:
 Run the CLI:
 
 ```bash
-# for vanilla React Native
+# Get walked through the prompts for the different options to start your new app
 npx ignite-cli new PizzaApp
-# to provide a custom bundle identifier (Android only):
-npx ignite-cli new PizzaApp --bundle=com.infinitered.pizzaapp
-```
 
-Ignite will walk you through the rest.
+# Accept all the recommended defaults and get straight to coding!
+npx ignite-cli new PizzaApp --y
```

Comment:
The `--y` vs `-y` confuses me a bit here (we chatted about this).

I looked around and this seems to be the place that the standard of `-y` vs `--yes` comes:

[https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html)

So let's do `--yes` and `-y` but not `--y`.

Reply:
I agree, the convention is the double dash is the more verbose name with the single dash for the shorthand notation

Reply:
Good catch, I forgot to update this. Commit is up now with a `-y` and `--yes` flag